tests/get_test.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+"""Test Get Storage bucket class."""
+import pytest
+from google.api_core.exceptions import NotFound
+from google.cloud.storage import Bucket
+from returns.pipeline import is_successful
+
+from storage_bucket.get import GetBucket, get_bucket
+
+
+class TestGetBucket(object):
+    """Test Get Bucket."""
+
+    def test_get_bucket_success(self):
+        """Create bucket, get Success Modal."""
+        bucket_result = GetBucket()(
+            storage_bucket_name='test-bucket-my-name-is',
```

Comment:
Please refer to tests/upload_test.py
add: 
STORAGE_BUCKET_NAME: Final[str] = os.getenv('STORAGE_BUCKET_NAME', 'not_set')
And use this env for all buckets that you are planning to actually find.
You can also overwrite this env variable with export when running your own tests. 

We set this env var in setup.cfg under tools pytest.